src/3p-frame.js

```diff
@@ -212,22 +212,30 @@ export function resetBootstrapBaseUrlForTesting(win) {
  * @return {string}
  */
 export function getDefaultBootstrapBaseUrl(parentWindow, opt_srcFileBasename) {
-  if (parentWindow.defaultBootstrapBaseUrl) {
-    return parentWindow.defaultBootstrapBaseUrl;
-  }
   const srcFileBasename = opt_srcFileBasename || 'frame';
```

Comment:
The function returns string, and was previously returning parentWindow.defaultBootstrapBaseUrl. So I presume that value /was/ a string. Now that value is an object. Just want to verify, was closure typing updated wherever that is declared?

Reply:
it still returns a string, but it's caching an object. I.e. it's returning `parentWindow.defaultBootstrapBaseUrl[srcFileBasename]` which is a string in the object `parentWindow.defaultBootstrapBaseUrl`

Reply:
Correct, still returning string, not returning object. My concern was just that parentWindow.defaultBootstrapBaseUrl has changed to an object, so was wanting to verify that there's not some now-deprecated comment somewhere still calling it a string. But my wording wasn't clear - thanks for catching it.